### ISSUE TITLE
whiteboard mgmt api

### ISSUE THREAD

**Issue by techsmyth** [alkem-io/whiteboard-collaboration-service#13]
## Description

As a sys admin I want to be able to manage whiteboard rooms

## Acceptance criteria

- [ ] api on whiteboard collaboration service that allows for a) listing all active rooms, with detailed information per room including who is connected, last activity, size of whiteboard, ... (what else?) b) being able to delete a room
- [ ] api to not be exposed outside of the cluster (so would need to port forward)
- [ ] api to be protected by PLATFORM_ADMIN privilege as issued to authenticated Alkemio users

**Comment by hero101** [comment]
Blocked until I am back

**Comment by hero101** [comment]
@SimoneZaza I don't believe the issue belongs to this Epic - I suggest doing this outside of an Epic, since we have already worked on the issue and it needs to be finalized.
